### DISCUSSION TITLE
Save view doesn't save window size

### DISCUSSION THREAD

**Discussion by StuartCT** [Stellarium/stellarium#2444] [General]
When I start Stellarium it only fills part of my screen (I have selected not to have full screen view so I can still see my Windows toolbar). So I then clicked the 'maximise' button to make it fill all of my screen (apart from the toolbar). Then I went to **"Configuration window" > Save view** 

But next time I open Stellarium, it's not saved the view as maximised.

How can I fix that?

**Comment by gzotti** [comment]
Window size is a _setting_, not a _view_ (orientation/field of view).

**Reply by StuartCT** [reply]
Thanks. So how can I save my window size? (I don't have this problem with any other windows application)

**Reply by gzotti** [reply]
On the dialog where you tried to save the window size setting by clicking "Save view", you find another button next to it labeled "Save settings". This is the button to press to save settings. More details can be found in our User Guide.

**Reply by StuartCT** [reply]
Perfect! That worked. Thank you
